library/include/G4LatticeLogical.hh

```diff
@@ -38,6 +38,7 @@
 //		precompute valley inverse transforms
 // 20200608  Fix -Wshadow warnings from tempvec
 // 20210919  M. Kelsey -- Allow SetVerboseLevel() from const instances.
+// 20231017  E. Michaud -- Add 'AddValley(const G4ThreeVector&)' to compute rotation matrix from valley's direction
```

Comment:
Please break comments at less than 80 columns.  See the lines above for examples.